gce/image/scylla_gce.json

```diff
@@ -38,9 +38,14 @@
       "source": "files/",
       "type": "file"
     },
+    {
```

Comment:
I think that the lines above no longer exists , we should remove them, right? and i think it's the same for all cloud providers

Reply:
@yaronkaikov Which lines you are mentioning about? "files/" ? If you are talking about "files/", it requires for --localrpm / --localdeb. If we only use --repo, we don't need it though.

Reply:
https://jenkins.scylladb.com/view/master/job/scylla-master/job/releng-testing/job/ami/651/console

Reply:
@yaronkaikov Ah, okay I found the bug. When I removed scylla_install_* scripts from files/ directory, I didn't added .gitkeep file, so git thinks files/ directory no longer exists.
I pushed new version that can keep files/ directory.